### DISCUSSION TITLE
Option to hide already downloaded beatmaps in osu!direct like in stable

### DISCUSSION THREAD

**Discussion by Stoppedpuma** [ppy/osu#18026] [Ideas] [answered]
Option to hide already downloaded beatmaps in osu!direct like how stable osu!direct has the show already downloaded beatmaps feature.
![example](https://user-images.githubusercontent.com/58333920/166089503-fb9f92f5-48bb-4e60-8e31-7bfb936f9bc0.png)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/15727
